src/dmd/id.d

```diff
@@ -327,6 +327,7 @@ immutable Msgtable[] msgtable =
     { "startaddress" },
     { "crt_constructor" },
     { "crt_destructor" },
+    { "Pctfe", "ctfe" },
```

Comment:
`Pctfe` is a bad name

Reply:
any suggestions? "ctfe" is already used for "__ctfe"

Reply:
Going for `pragmaCtfe` currently